Adds gradient to .gradient-text (#538)
microsoft/atlas-design: site/src/components/gradients.md

## Text color gradients

> Gradients transitions take up the entire width of a particular element, it's recommended to highlight inline elements, icons, or a portion of a heading, not the entire heading itself.

Reviewer: ```suggestion
Because gradient transitions take up the entire width of a particular element, it's recommended to highlight inline elements, icons, or a portion of a heading, and not the entire heading itself.
```

Author: Thanks, combined with @homing1 punctuation feedback.

> Gradients transitions take up the entire width of a particular element, it's recommended to highlight inline elements, icons, or a portion of a heading, not the entire heading itself.

Reviewer 2: ```suggestion
Gradients transitions take up the entire width of a particular element. It's recommended to highlight inline elements, icons, or a portion of a heading, not the entire heading itself.
```

Author: Thanks, combined with @BenDMyers copy feedback.

> Because gradient transitions take up the entire width of a particular element. It's recommended to highlight inline elements, icons, or a portion of a heading, and not the entire heading itself.

Reviewer: ```suggestion
Because gradient transitions take up the entire width of a particular element, it's recommended to highlight inline elements, icons, or a portion of a heading, and not the entire heading itself.
```

Reviewer: Please let commenters resolve their feedback.

In this case, combining two bits of grammar feedback led to something that also isn't grammatically correct - this should be one combined sentence, not split up like this.

Reviewer 2: This new version makes it one sentence, so I agree with Ben's suggestion.

Author: Sorry on my old team the author resolved the feedback after it was addressed. I've updated this text.

| base class name             | interpolated value     |
| --------------------------- | ---------------------- |
| `gradient-text-<colorname>` | `purple-blue`, `vivid` |

```html
<h3 class="font-size-h3 font-weight-bold">

> 	A dark gradient from <span class="gradient-text-purple-blue">purple to blue</span>

Reviewer: This gradient is theme-aware, and I'm not sure "dark" quite applies in all themes. Maybe "muted"?

```suggestion
	A muted gradient from <span class="gradient-text-purple-blue">purple to blue</span>
```

Author: Makes sense, updated.
